src/test/utilities/fakeAwsContext.ts

```diff
@@ -13,13 +13,22 @@ export const DEFAULT_TEST_ACCOUNT_ID = '123456789012'
 export const DEFAULT_TEST_PARTITION_ID = 'partitionQwerty'
 export const DEFAULT_TEST_REGION_CODE = 'regionQuerty'
 export const DEFAULT_TEST_REGION_NAME = 'The Querty Region'
+export const DEFAULT_TEST_DNS_SUFFIX = 'amazon.com'
```

Comment:
If this was blatantly fake data (like the above test values), would it break anything? 
The reason for asking is to prevent a false positive in testing.